src/actions/firebase/firebase.ts

```diff
@@ -0,0 +1,167 @@
+	import * as Hub from "../../hub"
+	import * as admin from 'firebase-admin';
+	import * as app from 'firebase-admin/app';
+
+	export class FirebaseAction extends Hub.Action {
+
+	  name = "firebase"
+	  label = "Firebase"
+	  iconName = "airtable/airtable.png"
+	  description = "Use firebase to send push notifications to mobile."
+	  storageBucket = 'gs://looker-9d4d2.appspot.com'
+	  contentType = "image/jpeg"
+	  notificationOptions = {
+      priority: "high",
+      timeToLive: 60 * 60 * 24,
+      contentAvailable: true,
+      mutableContent: true
+	  }
+	  static firebaseAdmin: app.App
+	  params = []
+	  supportedActionTypes = [Hub.ActionType.Query]
```

Comment:
is this true ? I think this action does not support query action type

Reply:
We need to support at least one action type
The other is dashboard and it looks like alerts does send dashboard query type.
I may have to verify this.

Reply:
can we instead add a new querytype enum "NONE" and use that ?
```
	  supportedActionTypes = [Hub.ActionType.None]
```

Reply:
@phillipperalez would this make more sense?

Reply:
None type would be fine and work for this case if we don't want it to show up in the scheduler modal

Reply:
Just checking if this was validated?
```
The other is dashboard and it looks like alerts does send dashboard query type.
I may have to verify this.
```

Reply:
Me and Aatish discussed that using None would mean that helltool should also send None, but that would lead to changes in existing alert code (which means slack will also need change)
So let's not use `None`